lib/semantics/mod-file.cc

```diff
@@ -173,6 +173,15 @@ void ModFileWriter::PutSymbol(
             }
             decls_ << '\n';
           },
+          [&](const CommonBlockDetails &x) {
+            PutLower(decls_ << "common/", symbol);
+            char sep = '/';
```

Comment:
Not the most useful feedback regarding your PR, but just because I saw it: `char sep{'/'};`